sale_report_delivered_subtotal/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Report Delivered subtotal",
+    "version": "11.0.1.0.0",
+    'author': 'Tecnativa,'
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/sale-reporting',
+    "category": "Sales",
+    "license": "AGPL-3",
+    "depends": [
+        "sale",
```

Comment:
IMHO this modules should depends in sale_management. Without sale_management you cannot use this module...

Reply:
I'm not completely sure about it. Isn't the rule to pull the minimum dependencies?

Reply:
Yes, you shouldn't put not technically needed dependencies.